app/services/clean_html.rb

```diff
@@ -0,0 +1,23 @@
+class CleanHtml
```

Comment:
could you move this to Feeds::CleanHtml or something like that? I want to eventually refactor `RssReader::Assembler` into something more self explanatory and with less responsibilities and since this is part of that machinery, I wonder if we should start by namespacing this correctly

Reply:
Yes, that makes sense. My main concern so far was moving things out of `app/labor`, but we might as well properly sort them while we're at it.